openmetadata-service/src/main/java/org/openmetadata/service/migration/utils/v180/MigrationUtil.java

```diff
@@ -24,4 +31,55 @@ public static void addCertificationOperationsToPolicy(CollectionDAO collectionDA
         List.of(MetadataOperation.EDIT_CERTIFICATION),
         collectionDAO);
   }
+
+  public static void addDenyDisplayNameRuleToBotPolicies(CollectionDAO collectionDAO) {
```

Comment:
again @aji-aju you are hard-coding these things. It should be part of the policy in itself, to deny displayName edit